docs/content/en/options-reference.md

```diff
@@ -186,6 +186,18 @@ Set this to `true` when using different domains for each locale. If enabled, no
 
 Whether [custom paths](/routing#custom-paths) are extracted from page files using babel parser.
 
+## `prefixAndDefaultRules`
+
+- type: `object`
+- default: `{ routing: 'default', switchLocale: 'default' }`
```

Comment:
Is it necessary to have separate settings for `routing` and `switchLocale`?

Reply:
I agree, it's better to keep only one property

Reply:
I agree, let's keep a single property for the behavior